packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts

```diff
@@ -645,23 +652,35 @@ export class ImageMode
     }
 
     renderable.userData.receiveTime = receiveTime;
-    renderable.setImage(image, /*resizeWidth=*/ undefined, (size) => {
+    renderable.setImage(image, /*resizeWidth=*/ undefined, () => {
       if (this.#fallbackCameraModelActive()) {
-        this.#updateFallbackCameraModel(size, getFrameIdFromImage(image));
+        this.#updateFallbackCameraModelFor(renderable);
+        this.#updateViewAndRenderables();
       }
     });
   };
 
-  #updateFallbackCameraModel = (size: { width: number; height: number }, frameId: string): void => {
-    const cameraInfo = createFallbackCameraInfoForImage({
-      frameId,
-      height: size.height,
-      width: size.width,
-      focalLength: DEFAULT_FOCAL_LENGTH,
-    });
-    this.#updateCameraModel(cameraInfo);
-    this.#updateViewAndRenderables();
-  };
+  /** Creates a fallback camera model based off of the renderable with a decoded image and updates the camera.
+   * Will no-op if there is not a decodedImage on the renderable.
+   * Be sure to call `#updateViewAndRenderables` after calling this method to update the camera and renderable.
+   */
+  #updateFallbackCameraModelFor(renderable: ImageRenderable) {
```

Comment:
```suggestion
  #updateFallbackCameraModel(renderable: ImageRenderable) {
```

the `For` suffix is weird – reads like Obj-C or Swift 🤣 

```swift
func updateFallbackCameraModel(for renderable: ImageRenderable) {
```

Reply:
ah yeah, I should just use the old method name. I had them both originally, but now they're consolidated it probably makes sense not to name it this way